src/NotepadNext/dialogs/MainWindow.cpp

```diff
@@ -1369,7 +1356,10 @@ void MainWindow::moveFileToTrash(ScintillaNext *editor)
 
     if (askMoveToTrash(filePath)) {
         if (editor->moveToTrash()) {
-            closeByPath(filePath);
+            closeCurrentFile();
```

Comment:
Why are you closing current file? One could open one file and delete another one.

Reply:
You are right! Good catch! 😄 This is what I get for trying to modify it on little sleep.